### PR TITLE
introduce MCAP CLI tool during installation just as a reference.

### PR DESCRIPTION
**Pull request by fujitatomoya**
address https://github.com/eProsima/DDS-Record-Replay/issues/109

### CODE REVIEW

File: docs/rst/developer_manual/installation/sources/linux.rst [hunk 1]
```diff
@@ -217,6 +217,10 @@ For example, on Ubuntu use the command:
 
    sudo apt install liblz4-dev libzstd-dev
 
+.. note::
+
+    To work with MCAP files via command line interface, you can use `MCAP CLI <https://mcap.dev/guides/cli>` _ to manage the data in MCAP files.
```
Comment (fujitatomoya) [inline docs/rst/developer_manual/installation/sources/linux.rst:222]: i thought we do not want to `MCAP CLI` usage and details in eProsima doc here for the maintenance. instead, we can introduce that there is `MCAP CLI` that we can use to manage the saved MCAP files via command line interface here.

**Comment by fujitatomoya** [comment]
@juanlofer-eprosima can you take a look when you have time?

**Review by rsanchez15 — APPROVED** [review]

**Review by rsanchez15 — CHANGES_REQUESTED** [review]
Hi @fujitatomoya, could you rebase this to `main` branch? Then we should be able to merge it.

**Comment by fujitatomoya** [comment]
@rsanchez15 done rebase against upstream/main.

**Pull request merged**